### ISSUE TITLE
Staging Review finding: Remove this provider improper focus management

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#111496]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Unified Appointment Experience
**Product:** Appointments
**Feature:** Community Care Direct Scheduling

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @rsmithadhoc (Accessibility)

### Description

When the user removes the provider via the modal, focus is moved to the Veterans Crisis Line link in the page footer instead of being returned to where they activated the modal.

## Recommended action

Return focus to the Choose a provider button.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 1.4.13 Content on Hover or Focus - Level AA](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by rsmithadhoc** [comment]
> [@rsmithadhoc](https://github.com/rsmithadhoc) could you provide a screenshot of the interaction with the issue? TY!

@outerpress Added screenshot and gif of how to get there, thanks!

**Comment by outerpress** [comment]
Perfect, TY @rsmithadhoc !

For the delivery team and @kristenmcconnell @klawyer : 

- The CC request flow was originally designed to support selecting more than one provider
- However, that functionality was never added, so only one provider is selectable
- Do we want to add that functionality in the future
- If not, we could remove this interaction entirely, which has some other issues that are coming up in the a11y review
- Instead, a user could just select from a radio button list of providers

@Dottifacilities @a6marinos @StevenStraily

**Comment by kristenmcconnell** [comment]
This only occurs during the community care appointment request workflow and not direct scheduling, so this component wasn't updated/touched as part of this community care effort. We will tackle this in the above ticket that Peter linked, but will be prioritizing the tickets affecting community care direct scheduling, so that feature can release first.
